src/com/sun/jna/NativeLibrary.java

```diff
@@ -947,6 +947,9 @@ else if (Platform.isARM()) {
             cpu = "arm";
             libc = "-gnueabi";
         }
+        else if (Platform.isMIPS()) {
+            cpu = (Platform.is64Bit() ? "mips64el" : "mipsel");
+        }
```

Comment:
I checked the path of libc.so for example,
Fedora (mips64el) : /usr/lib64/libc.so
Fedora (mips32el) : /usr/lib32/libc.so
Debian (mips64el): /usr/lib/mips64el-linux-gnuabi64/libc.so
Debian (mipsel): /lib/mipsel-linux-gnu/libc.so.6
Debian (mips): /lib/mips-linux-gnu/libc.so.6
The path under the situation of Fedora is set by:
`"/usr/lib" + archPath`

So on debian, I think you suggestion is correct. However, I think it should be
`libc = "-gnuabi64"; `
rather than
`libc = "-gnueabi64";`